client-app/components/atoms/select/select.vue

```diff
@@ -0,0 +1,199 @@
+<template>
+  <div>
+    <div>
```

Comment:
Please, add `v-if="label"` here to allow using unlabeled selects (ex: sort on category page)